solver/src/solver/paraswap_solver/api.rs

```diff
@@ -185,6 +188,7 @@ impl TransactionBuilderQuery {
             .expect("unexpectedly invalid URL segment");
         url.query_pairs_mut().append_pair("skipChecks", "true");
 
+        tracing::debug!("Paraswap API (transaction) query url: {}", url);
```

Comment:
Isn't this double logging the TransactionBuilderQuery?

Reply:
This is a POST request, so the url is very short. It's there so that we don't need to find the URL of the api if we want to simulate a query after seeing it from the logs.